app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from os import makedirs, path, remove, replace
+
+
+class ErrorCommand(Exception):
+    pass
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        mv, source_file, destination_file = command.split()
```

Comment:
Don't use `command.split()` twice. Create a separate variable for storing result of `command.split()` and use it where you need.